RafRate.API/Extentions/AppExtentions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Diagnostics;
+using System.ComponentModel.DataAnnotations;
+
+namespace RafRate.API.Extentions
+{
+    public static class AppExtentions
```

Comment:
It is a bad name for the class where you only add exception handler

Reply:
If I sew this in code, I didn't understand what is it and why this handler in this class